utils/notifications_utils/template.py

```diff
@@ -38,11 +38,12 @@
 from notifications_utils.template_change import TemplateChange
 
 
-template_env = Environment(loader=FileSystemLoader(
-    path.join(
-        path.dirname(path.abspath(__file__)),
-        'jinja_templates',
-    )
+template_env = Environment(autoescape=select_autoescape(['html', 'htm', 'xml']),
+    loader=FileSystemLoader(
+        path.join(
+            path.dirname(path.abspath(__file__)),
+            'jinja_templates',
+        )
```

Comment:
LGTM. Can't see any obvious issues but I didn't test every page or function.

Reply:
We'll test in production 😄